dts/bindings/watchdog/arm,cmsdk-watchdog.yaml

```diff
@@ -14,3 +14,8 @@ properties:
 
     reg:
       category: required
+
+    clocks:
+      type: array
```

Comment:
Now that clocks is in base.yaml, we just need 'category: required' here the type, and description aren't needed.

Reply:
Done